max_linear_vel       : 1.1
max_angular_vel      : 3.14
limit_vel_rate_right : 1.08
limit_vel_rate_left  : 1.05
limit_vel_step_right : 100
limit_vel_step_left  : 100
gain_p_right         : 0.1775
gain_i_right         : 0.0380
gain_p_left          : 0.1850
gain_i_left          : 0.03335

